runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -276,8 +276,8 @@ private Dataflow buildMockDataflow() throws IOException {
     when(mockProjects.locations()).thenReturn(mockLocations);
     when(mockLocations.jobs()).thenReturn(mockJobs);
     when(mockJobs.create(eq(PROJECT_ID), eq(REGION_ID), isA(Job.class))).thenReturn(mockRequest);
-    when(mockJobs.list(eq(PROJECT_ID), eq(REGION_ID))).thenReturn(mockList);
-    when(mockList.setPageToken(anyString())).thenReturn(mockList);
+    when(mockJobs.list(any(), any())).thenReturn(mockList);
```

Comment:
This particular line seems like it does not faithfully maintain the test.

Reply:
Apparently the test in question was passing `null` here instead of the required parameters. I will check again why this particular line was wrong.

Reply:
Fixed.